StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1618UnitTests.cs

```diff
@@ -0,0 +1,291 @@
+﻿namespace StyleCop.Analyzers.Test.DocumentationRules
+{
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Analyzers.DocumentationRules;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using TestHelper;
+    using Xunit;
+
+    /// <summary>
+    /// This class contains unit tests for <see cref="SA1618GenericTypeParametersMustBeDocumented"/>.
+    /// </summary>
+    public class SA1618UnitTests : CodeFixVerifier
```

Comment:
:bulb: Test suites that don't include code fixes should extend `DiagnosticVerifier` instead of `CodeFixVerifier`.